db/re/item_db.txt

```diff
@@ -8336,8 +8336,8 @@
 14166,F_Giant_Fly_Wing_Box,Giant Fly Wing Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12212,10; },{},{}
 14167,F_Neuralizer_Box,Neuralizer Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12213,1; },{},{}
 14168,F_Convex_Mirror_Box,Convex Mirror Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12214,10; },{},{}
-14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,10; },{},{}
-14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,10; },{},{}
+14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,20; },{},{}
+14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,20; },{},{}
```

Comment:
kRO description give 10 scrolls
```suggestion
14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,10; },{},{}
```

Reply:
Right, I've also confirmed it on Aegis, I'll revert it back